packages/db/src/types.ts

```diff
@@ -41,12 +42,27 @@ export interface PostgresCollectionRecord extends PostgresCollection {
   updated_at: Date,
 }
 
-export interface ExecutionRecord {
+export interface PostgresExecution {
   arn: string,
   async_operation_cumulus_id?: number,
   collection_cumulus_id?: number,
   parent_cumulus_id?: number,
-  cumulus_version: string,
+  cumulus_version?: string,
+  url?: string,
+  status?: ExecutionRecordStatus,
+  tasks?: Object, // TODO need specific type?
+  error?: Object, // TODO need specific type?
```

Comment:
It looks like these are just JSON objects. Should I have different types for them though? Are there keys that these always have?